Formula/aws-sso-cli.rb

```diff
@@ -0,0 +1,22 @@
+class AwsSsoCli < Formula
+  desc "Securely manage AWS API credentials using AWS SSO"
+  homepage "https://github.com/synfinatic/aws-sso-cli"
+  url "https://github.com/synfinatic/aws-sso-cli.git",
+      tag:      "v1.9.10",
```

Comment:
Looks like this release did not get marked as latest release yet?
https://github.com/synfinatic/aws-sso-cli/releases

Reply:
Correct.  It's tagged, but not done as a full blown release, because this release is only for getting aws-sso into homebrew core and didn't want to spam watchers with another release.

Reply:
If it's not a release we can't merge it yet

Reply:
okay... there's your release.

Reply:
Thanks!